packages/website/components/account/addPaymentMethodForm/addPaymentMethodForm.js

```diff
@@ -52,12 +52,12 @@ const AddPaymentMethodForm = ({ setHasPaymentMethods, setEditingPaymentMethod, c
         await userBillingSettings(paymentMethod.id, currStorageSubscription);
         setHasPaymentMethods?.(true);
         setEditingPaymentMethod?.(false);
-        setPaymentMethodError('');
+        setPaymentMethodError(null);
       } catch (error) {
-        let message;
-        if (error instanceof Error) message = error.message;
-        else message = String(error);
-        setPaymentMethodError(message);
+        if (!(error instanceof APIError)) {
```

Comment:
will Sentry see these?  if not, do we want to explicitly send them to Sentry?

Reply:
We don't have sentry at all on the frontend! but I want to (search slack dag.house)... will handle in separate issue/priority. Good idea/concern though :)